Add `osu! Vintage Map Tourney!` tournament articles (#5959)
ppy/osu-wiki: wiki/Tournaments/OVMT/2/en.md

---
tags:
  - OVMT
  - OVMT2
---

# osu! Vintage Map Tourney 2! Electric Boogaloo!

> The **osu! Vintage Map Tourney 2! Electric Boogaloo!** (***OVMT2***) was a 1v1 single-elimination osu! tournament hosted by ![][flag_GB] [mmi-](https://osu.ppy.sh/users/5444914). It was the second instalment of osu! Vintage Map Tourney!.

Reviewer: ```suggestion
The **osu! Vintage Map Tourney 2! Electric Boogaloo!** (***OVMT2***) was a 1v1 single-elimination osu! tournament hosted by ![][flag_GB] [mmi-](https://osu.ppy.sh/users/5444914). It was the second instalment of the osu! Vintage Map Tourney!.
```

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2017-05-30/2017-06-20 |
| Round of 128 | 2017-06-21/2017-06-27 |
| Round of 64 | 2017-06-28/2017-07-04 |
| Round of 32 | 2017-07-05/2017-07-11 |
| Round of 16 | 2017-07-12/2017-07-18 |
| Quarterfinals | 2017-07-19/2017-07-25 |
| Semifinals | 2017-07-26/2017-08-01 |
| Finals | 2017-08-08 & 2017-09-01 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 1 year of osu!supporter, profile badge |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 1 year of osu!supporter |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 6 months of osu!supporter |

![](img/badge.png "OVMT2 winner badge")

## Organisation

osu! Vintage Map Tourney! was run by various community members.

| Position | Member(s) |
| :-- | :-- |
| Host | ![][flag_GB] [mmi-](https://osu.ppy.sh/users/5444914) |
| Mappool selector | ![][flag_GB] [mmi-](https://osu.ppy.sh/users/5444914) |
| Streamer | ![][flag_GB] [mmi-](https://osu.ppy.sh/users/5444914), ![][flag_FI] [LefafeI](https://osu.ppy.sh/users/2295850) |

> | Commentators | ![][flag_GB] [mmi-](https://osu.ppy.sh/users/5444914), ![][flag_FI] [LefafeI](https://osu.ppy.sh/users/2295850), ![][flag_US] [Starrodkirby86](https://osu.ppy.sh/users/410) |

Reviewer: ```suggestion
| Commentator | ![][flag_GB] [mmi-](https://osu.ppy.sh/users/5444914), ![][flag_FI] [LefafeI](https://osu.ppy.sh/users/2295850), ![][flag_US] [Starrodkirby86](https://osu.ppy.sh/users/410) |
```

  - [System of a Down - Vicinity of Obscenity (Larto) [Impossible]](https://osu.ppy.sh/beatmapsets/13768#osu/50712)
  - [Boots Randolph - Yakety Sax (Mashley) [Ridiculous]](https://osu.ppy.sh/beatmapsets/17943#osu/63804)
  - [Tachibana Miya - Miya to Tengoku to Jigoku (AngelHoney) [Hentai]](https://osu.ppy.sh/beatmapsets/17450#osu/62269)
  - [Renard - Banned Forever (Blue Dragon) [Nogard]](https://osu.ppy.sh/beatmapsets/16349#osu/64267)
  - [Kucchy vs Akky - Yakumo \~ JOINT STRUGGLE (DJPop) [SOLO]](https://osu.ppy.sh/beatmapsets/12909#osu/48098)
- FreeMod
  - [YMCK - Family Dondon (Zekira) [Oni]](https://osu.ppy.sh/beatmapsets/12190#osu/45923)
  - [Ochiai Yurika - Koibumi2000 (arken1015) [Oni(Futsuu)]](https://osu.ppy.sh/beatmapsets/10539#osu/46298)
  - [IOSYS - Usatei (Card N'FoRcE) [RUN!!]](https://osu.ppy.sh/beatmapsets/3959#osu/22993)
- ForceMod
  - [Dragonforce - Fury Of The Storm (lesjuh) [Kingcobra's Expert]](https://osu.ppy.sh/beatmapsets/10460#osu/41646)
  - [Amuro vs Killer - Mei (v2b) [Another]](https://osu.ppy.sh/beatmapsets/8965#osu/36290)
  - [nam5 - Heavenly Shake (Kite) [Extra Stage]](https://osu.ppy.sh/beatmapsets/20892#osu/72783)

> - TieBreaker

Reviewer: ```suggestion
- Tiebreaker
```

  - [mforestpireo - Kanojo Ga Ichiban Shoujo Nanoka? (Teara) [KIRBY Mix]](https://osu.ppy.sh/beatmapsets/11773#osu/50845)
  - [RevenG - Sakura (Kharl) [Insane]](https://osu.ppy.sh/beatmapsets/4638#osu/25296)
  - [DJ Sharpnel - StrangeProgram (happy30) [HappyMiX]](https://osu.ppy.sh/beatmapsets/5774#osu/28065)
  - [TAKA respect for J.S.B - Ubertreffen (kiddly) [Another]](https://osu.ppy.sh/beatmapsets/15740#osu/56830)
  - [Susumu Hirasawa - Aurora 2 (NachoPiggy) [KIRBY Mix]](https://osu.ppy.sh/beatmapsets/10978#osu/49206)
  - [07th Expansion - rog-limitation (AngelHoney) [Insane]](https://osu.ppy.sh/beatmapsets/14994#osu/54581)
- FreeMod
  - [Dj Command - SMooooch (Akiba Kobo Mix) (Natteke) [Collab]](https://osu.ppy.sh/beatmapsets/21342#osu/74117)
  - [Seiryu - 3y3s (yeahyeahyeahhh) [Another]](https://osu.ppy.sh/beatmapsets/15241#osu/55437)
- ForceMod
  - [COOL&CREATE - Saishoukichiku Imouto Flandre S (dksslqj) [Lunatic]](https://osu.ppy.sh/beatmapsets/14853#osu/54145)
  - [SUPER STAR -MITSURU- - She is my wife (yeahyeahyeahhh) [Another]](https://osu.ppy.sh/beatmapsets/22331#osu/77110)

> - TieBreaker

Reviewer: ```suggestion
- Tiebreaker
```

  - [DCX - Flying High (DJ Splash Remix) (yeahyeahyeahhh) [InoSane]](https://osu.ppy.sh/beatmapsets/22194#osu/76663)
  - [IOSYS - Kanbu de Tomatte Sugu Tokeru \~ Kyouki no Udongein (ouranhshc) [Lunatic]](https://osu.ppy.sh/beatmapsets/12292#osu/61450)
  - [dBu - Shinkou Fuuka Kyoku \~ Native Faith (saymun) [Lunatic]](https://osu.ppy.sh/beatmapsets/17972#osu/63893)
  - [The Prodigy - We Are The Ruffest (whymeman) [Ruffest]](https://osu.ppy.sh/beatmapsets/9538#osu/38007)
  - [Susumu Hirasawa - Amor Buffer (Real1) [KIRBY Mix]](https://osu.ppy.sh/beatmapsets/11702#osu/44526)
  - [DJ Sharpnel - Exciting Hyper Highspeed Star (xBubu) [Highspeed]](https://osu.ppy.sh/beatmapsets/16619#osu/62589)
- FreeMod
  - [IOSYS - Alice wa Taihen na Kashi Wo Urooboe Deshita (Zekira) [Enasni]](https://osu.ppy.sh/beatmapsets/11076#osu/42697)
  - [Seiryu X Donald - Time to Donald (James) [Another]](https://osu.ppy.sh/beatmapsets/6950#osu/30613)
- ForceMod
  - [IOSYS - Poinsettia (Aakiha) [Lunatic]](https://osu.ppy.sh/beatmapsets/18382#osu/65233)
  - [Cascada - What Hurts The Most (Nightcore Remix) (Lunah) [jericho2442's Pain!]](https://osu.ppy.sh/beatmapsets/20239#osu/73561)

> - TieBreaker

Reviewer: ```suggestion
- Tiebreaker
```

  - [Toyosaki Aki - Happy!? Sorry!! (DJPop) [Insane]](https://osu.ppy.sh/beatmapsets/6892#osu/30481)
  - [Mr. - Hats, Gloves and Overalls (Hikato) [Doomsday's Insane]](https://osu.ppy.sh/beatmapsets/7305#osu/31838)
  - [Demetori - Innocent Treasures (fartownik) [SteRRuM's Treasure]](https://osu.ppy.sh/beatmapsets/15393#osu/56283)
  - [Sound Holic - Optical Cellophane (PandaCath) [KIRBY Mix]](https://osu.ppy.sh/beatmapsets/6704#osu/31050)
  - [Kugimiya Rie - Pre-Parade (Instrumental) (LKs) [H-boy's Hard]](https://osu.ppy.sh/beatmapsets/14563#osu/53440)
  - [IOSYS - Princess Party \~Seishun Kinshi Rei\~ (DJPop) [Insane]](https://osu.ppy.sh/beatmapsets/14294#osu/52386)
- FreeMod
  - [ZUN - Taketorihisyou \~ Lunatic Princess (no-7) [Lunatic]](https://osu.ppy.sh/beatmapsets/13220#osu/54363)
  - [Nightcore - You Got Me Dancing (osuplayer111) [Insane]](https://osu.ppy.sh/beatmapsets/15917#osu/57380)
- ForceMod
  - [COOL&CREATE - Rapid Ensemble (Doomsday93) [Lunatic]](https://osu.ppy.sh/beatmapsets/13235#osu/49067)
  - [Saitama Saisyu Heiki - Thunder Light (-\_Kite\_-) [Insane]](https://osu.ppy.sh/beatmapsets/13103#osu/55059)

> - TieBreaker

Reviewer: ```suggestion
- Tiebreaker
```

  - [Caramell - Caramelldansen (Speedycake Remix) (0\_o) [LESJUH DIFFICULTY!!]](https://osu.ppy.sh/beatmapsets/7380#osu/43423)
  - [Capcom Sound Team - Dr. Wily Stage 1-2 (Cyclone) [Insane]](https://osu.ppy.sh/beatmapsets/4115#osu/29802)
  - [OK Go - It's a Disaster (Larto) [Insane]](https://osu.ppy.sh/beatmapsets/22128#osu/76630)
  - [MiYAMO - Kapanet Nitori (NachoPiggy) [Overpriced]](https://osu.ppy.sh/beatmapsets/16606#osu/59420)
  - [Niko - Made of Fire (lesjuh) [Heavy]](https://osu.ppy.sh/beatmapsets/10112#osu/40034)
  - [Asumi Kana - SOMEONE ELSE (miccoliasms) [Insane]](https://osu.ppy.sh/beatmapsets/13833#osu/50907)
  - [Susumu Hirasawa - Switched-On Lotus (Real1) [KIRBY Mix]](https://osu.ppy.sh/beatmapsets/10977#osu/43827)
- FreeMod
  - [jun - KIMONO <3 PRINCESS (ouranhshc) [Insane]](https://osu.ppy.sh/beatmapsets/19691#osu/69102)
  - [DJ YOSHITAKA-G feat. Michael a La Mode - Gold Rush (Chacha17) [Another]](https://osu.ppy.sh/beatmapsets/13879#osu/51159)
- ForceMod
  - [ClariS - Irony (TV Size) (osuplayer111) [Insane]](https://osu.ppy.sh/beatmapsets/22121#osu/76445)
  - [Nightcore - Pinball (jericho2442) [Madness O\_O]](https://osu.ppy.sh/beatmapsets/7930#osu/37433)

> - TieBreaker

Reviewer: ```suggestion
- Tiebreaker
```

  - [supercell - Kimi no Shiranai Monogatari (TV Size) (Alace) [Insane]](https://osu.ppy.sh/beatmapsets/11086#osu/42710)
  - [HTT - NO, Thank You! (TV Size) (Dangaard) [Insane]](https://osu.ppy.sh/beatmapsets/18156#osu/64468)
  - [Anamanaguchi - Rox 300 (Sushi) [Supreme Master]](https://osu.ppy.sh/beatmapsets/21471#osu/74494)
  - [Blink 182 - The Rock Show (ztrot) [Insane]](https://osu.ppy.sh/beatmapsets/15476#osu/56723)
  - [Y. Kaneoka, A. Nakatuka, K. Yamamoto - Training \~ Credits (Gens) [Insane]](https://osu.ppy.sh/beatmapsets/8206#osu/33990)
  - [Nico Nico Douga - Un-Tan-Tan de TeTeTe (Card N'FoRcE) [Insane]](https://osu.ppy.sh/beatmapsets/7447#osu/32230)
- FreeMod
  - [IOSYS - Miracle-Hinacle (-SiN-) [Lesjuh Curse]](https://osu.ppy.sh/beatmapsets/3609#osu/32329)
  - [Hashimoto Miyuki - Symphonic Love (digiaria) [\_loli\~p\~o\~p\~]](https://osu.ppy.sh/beatmapsets/16964#osu/64028)
- ForceMod
  - [Fantasy Project - Crazy Baby (Nightcore Mix) (Louis Cyphre) [pieguy's Hard]](https://osu.ppy.sh/beatmapsets/23415#osu/80199)
  - [Basshunter - Ievan Polkka Trance Remix (Teara) [ShiNexy]](https://osu.ppy.sh/beatmapsets/10406#osu/40563)

> - TieBreaker

Reviewer: ```suggestion
- Tiebreaker
```

  - [Trey Parker - Gay Fish (Zerostarry) [Intense]](https://osu.ppy.sh/beatmapsets/6526#osu/29519)
  - [Rossini & MC F 40 - Guillaume Tell (Sushi971) [Standard]](https://osu.ppy.sh/beatmapsets/11742#osu/44694)
  - [Outkast - Hey Ya! (lesjuh) [Hey Lesjuh!]](https://osu.ppy.sh/beatmapsets/10605#osu/42654)
  - [Hisaka Yoko - Listen!! (TV Size) (DJPop) [Hard]](https://osu.ppy.sh/beatmapsets/14701#osu/53700)
  - [Susumu Hirasawa - Shiroi Oka - Maromi no Theme (Beuchi-chan) [KIRBY Mix]](https://osu.ppy.sh/beatmapsets/16252#osu/58333)
  - [Galaxy Angel Rune's cast - Uchuu de Koi wa Rurun Ruuuun (Sushi971)](https://osu.ppy.sh/beatmapsets/4994#osu/27730)
- FreeMod
  - [m.o.v.e - Rage your dream (INITIAL D mix) (Card N'FoRcE) [Hard]](https://osu.ppy.sh/beatmapsets/17330#osu/61840)
  - [Oratorio The World God Only Knows - God only knows Dai San Maku (O.A. version) (DJPop) [Insane]](https://osu.ppy.sh/beatmapsets/22196#osu/77308)
- ForceMod
  - [Scognito - Spelunker (Kurai\_) [Fight69's Hard]](https://osu.ppy.sh/beatmapsets/16382#osu/60234)
  - [HTT - NO, Thank You! (TV Size) (Dangaard) [S i R i R u's Hard]](https://osu.ppy.sh/beatmapsets/18156#osu/64555)

> - TieBreaker

Reviewer: ```suggestion
- Tiebreaker
```

| **\[ Scarlet Red \]** ![][flag_US] | **3** | 2 | ![][flag_DE] srb |  |
| **Karthy** ![][flag_GB] | **3** | 2 | ![][flag_GB] Jimzzyyy |  |
| **Bubbleman** ![][flag_GB] | **3** | 2 | ![][flag_PL] Barto |  |
| Mrshadowfan98 ![][flag_CA] | 0 | 0 | ![][flag_FI] Cryption | *no-show* |
| **peng** ![][flag_CA] | **1** | 0 | ![][flag_PL] - Vanilla - | *forfeit* |
| Wnd ![][flag_US] | 0 | **3** | ![][flag_FR] **ElektroFire** |  |
| **crushsquids** ![][flag_US] | **1** | 0 | ![][flag_GB] Malteser | *forfeit* |

Monday, 26 June 2017:

| Player 1 | | | Player 2 |
| --: | :-: | :-: | :-- |
| djsettle ![][flag_GB] | 1 | **3** | ![][flag_FI] **Mara** |
| ExploXd ![][flag_CZ] | 0 | **3** | ![][flag_CZ] **Kaplan** |
| MicMck101 ![][flag_US] | 0 | **3** | ![][flag_GB] **Jerome** |

Tuesday, 27 June 2017:

| Player 1 | | | Player 2 | Match info |
| --: | :-: | :-: | :-- | :-- |
| **Joezapy** ![][flag_US] | **3** | 0 | ![][flag_GB] Doomsday Is Bad |  |
| **Ora** ![][flag_US] | **1** | 0 | ![][flag_IT] Rainfall | *forfeit* |

## Ruleset

- This tournament is a 1v1 single-elimination international osu! tournament.
- This tournament will consist of 128 players.

> - Your rank must be lower than #50,000 to play. Signups under this rank will be considered on a per player basis.

Reviewer 2: ```suggestion
- Your rank must be lower than #50,000 to play. Sign-ups under this rank will be considered on a per-player basis.
```

Author: ```suggestion
- Your rank must be better than #50,000 to play. Sign-ups worse this rank will be considered on a per-player basis.
```

- Match-ups for the first round will be randomly selected. No seeding, just purely random. If there are not enough people to fill the initial bracket, then the higher ranked players will advance to the next tier.

### Match info

- Each pool will consist of 8 NoMod, 2 FreeMod, 2 ForceMod and a TieBreaker map.
  - From Quaterfinals, each mappool will have an added 2 NoMod maps.
  - From Finals, each mappool will have an added FreeMod and ForceMod map.
  - On NoMod maps, both players must not use any mod.
  - On FreeMod maps, players are free to choose wether to use a mod or not. Every ranked mod is allowed.
    - If both players agree, DT or HT can be enabled for the map.
  - On ForceMod maps, players must pick a mod. Every ranked mod is allowed, excluding DT, SD, NF or SO.
  - The TieBreaker map follows FreeMod rules.
- The Round of 32 is Best of 5, every round afterwards is Best of 7, the Grand Final is Best of 9.
- Both players will be contacted by the match referee before the match starts in bancho chat. Players have until 15 minutes after the scheduled start time of the match to show up, or they automatically forfeit.

> - Players are able to play a warm up map if they so choose. Warmup maps should be under 4 minutes 30 seconds long and they must be submitted to osu.ppy.sh.

Reviewer 2: ```suggestion
- Players are able to play a warm-up map if they so choose. Warm-up maps should be under 4 minutes 30 seconds long and they must be submitted to osu.ppy.sh.
```
